bin/generate_unfiltered_sce.R

```diff
@@ -205,12 +207,14 @@ sample_type <- sample_metadata_df |>
     )
   ) |>
   dplyr::select(sample_id, sample_type) |>
-  # convert into named vector
-  tibble::deframe()
+  # convert into named list
+  tibble::deframe() |>
+  strsplit(split = ", ")
```

Comment:
Making a single character split.
```suggestion
  strsplit(split = ";")
```